content/en/_index.md

```diff
@@ -0,0 +1,219 @@
+---
```

Comment:
This PR seems to re-add `content/en/_index.md`. I moved over over to `content/en/_index.html` as we added a lot of HTML in the landing page and upon using a newer `docsy` version our generated HTML wasn't valid any more, etc. We can't have `_index.md` and `_index.html` for generating the same page.

Reply:
On juozasg:main branch I have deleted _index.md that I accidentally committed.